src/ert/shared/models/base_run_model.py

```diff
@@ -525,17 +525,26 @@ def _post_update_data(self, parent_ensemble_id: str, algorithm: str) -> str:
         return update_id
 
     def check_if_runpath_exists(self) -> bool:
+        """
+        Determine if the run_path exists by checking if it contains
+        at least one iteration directory for the iterations in the active mask.
+        The run_path ccan contain one or two %d specifiers ie:
```

Comment:
`can` is misspelled.

Reply:
The we also need to update the docs as they state 

`It should contain at least one %d specifier..`